spec/features/company_pages_spec.rb

```diff
@@ -0,0 +1,141 @@
+feature 'index page' do
+
+  context "not an admin" do
+    scenario 'not logged in' do
+      visit companies_path
+      expect(page).to have_content 'need to sign in'
+    end
+
+    scenario 'logged in as student' do
+      student = FactoryGirl.create(:student)
+      login_as(student, scope: :student)
+      visit companies_path
+      expect(page).to have_content 'You are not authorized to access this page'
+    end
+  end
+
+  context "as an admin" do
+    let(:cohort) { FactoryGirl.create(:cohort) }
+    let(:admin) { FactoryGirl.create(:admin) }
+    let!(:company) { FactoryGirl.create(:company) }
```

Comment:
The tests failed when I changed it.  I think you have to force rspec to evaluate it before you navigate to the index page, otherwise it won't be there.